### DISCUSSION TITLE
Audio and video are not synchronized, and audio is seriously lost

### DISCUSSION THREAD

**Discussion by xiuer-1573** [alvr-org/ALVR#1080] [Q&A]
I have modified the automatic bit rate adjustment strategy of the video part, resulting in a large amount of audio loss. How should I solve this problem?

**Comment by zmerp** [comment]
Can you show the changes you made?

**Reply by xiuer-1573** [reply]
This is not because of my modification, but because of packet loss in audio. I see FEC mechanism in video, may I ask if there is any strategy for retransmission or recovery in audio after packet loss? thank you

**Reply by zmerp** [reply]
No, there is no retransmission or recovery. If a packet arrives out of order it is discarded by StreamSocket, and if a packet is lost or discarded the audio implementation does a fade-out/in that is heard as a brief audio cut (this is to avoid annoying clicks). There is no major change in ALVR code/logic, but something may have changed when I moved audio backend recently. The commit that did these changes is d84d8e6e2717a51169fba8ef309979599efb3ebb (7 days ago). Please test before this commit if it solves your problem.
